shared/components/NavigationBar/index.scss

```diff
@@ -1,57 +1,150 @@
+          padding: 14px 0px 14px 80px;
+          display: block;
+          font-weight: bold;
+          font-size: 18px;
+          color: $conference-color;
+          text-decoration: none;
+          &:hover {
+            background: $conference-color;
+            color: $grey;
+          }
+        }
+        a.active {
+          background: $conference-color;
+          color: $grey;
+        }
+      }
     }
   }
 
-  &--Conference {
-    a,
-    a:visited {
-      color: $conference-color;
+  /* Opening menu from hamburger */
+  input.trigger {
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+    &:checked ~ ul {
+      @media (max-width: $tablet-width) {
+        display: block !important; /* Displaying the menu when the burger is clicked */
+        display: flex;
+        flex-direction: row;
+      }
     }
-
-    a:focus,
-    a:hover {
-      color: $conference-color-faded;
+    /* When the menu is opened, the hamburger label's text is changed to white */
+    &:checked ~ label {
+      @media (max-width: $tablet-width) {
+        color: white;
+      }
     }
-
-
-    a.NavigationBar__tab--active {
-      background-color: $conference-color-faded;
+    /* When the menu is opened, the hamburger icon is changed to white */
+    &:checked ~ label:before {
+      @media (max-width: $tablet-width) {
+        color: white;
+      }
     }
   }
+  label {
+    position: relative;
+    display: block;
+    width: 5px;
+    padding: 9px 0px 9px 35px;
+    font-size: 18px;
+    font-weight: bold;
+    margin: 0;
+    cursor: pointer;
+    line-height: 2em;
+    color: $conference-color;
+    margin-left: 44px;
+  }
+  /* Burger icon */
+  label:before {
+    position: absolute;
+    top: 8px;
+    left: 0;
+    content: "\2261";
+    font-weight: normal;
+    font-size: 30px;
+    color: $conference-color;
+  }
 
-  &__tabs {
+  .ticket__button__container {
+    position: absolute;
+    right: 44px;
+    top: 12px;
     display: flex;
-    flex-flow: row wrap;
+    justify-content: center;
     align-items: center;
-    margin: 0;
-    padding: 0;
-
-    a {
-      $border-bottom-size: 3px;
-
+    .ticket__button {
+      background-color: red;
+      border-radius: 6px;
+      text-transform: uppercase;
+      background-color: #d2091e;
+      color: #fff;
+      cursor: pointer;
+      text-decoration: none;
+      border-radius: 6px;
+      text-transform: uppercase;
+      border: 0;
+      padding: 5px 15px;
+      color: #fff;
+      font-weight: 700;
       text-decoration: none;
-      font-weight: bold;
-      font-size: 20px;
-      width: 40vw;
-      min-width: 130px;
       text-align: center;
-      height: 50px;
-      vertical-align: middle;
-      display: table-cell;
-      padding-bottom: $border-bottom-size;
+    }
+  }
 
-      &.NavigationBar__tab--active {
-        color: $grey;
-        border-bottom: $border-bottom-size solid $grey;
-        padding-bottom: 0;
+  /* Tablet and Desktop View */
```

Comment:
I feel the media query is clear enough without a comment thanks to the variable :)